src/utilities/utilities.hpp

```diff
@@ -28,6 +28,21 @@
 #include "utilities/clblast_exceptions.hpp"
 #include "utilities/msvc.hpp"
 
+#define FULL_SEARCH_STRATEGY 0
+#define RANDOM_SEARCH_STRATEGY 1
+#define PSO_STRATEGY 2
+#define DVDT_STRATEGY 3
+
+#define DEFAULT_STRATEGY 0
+
+#define DEFAULT_PSO_SWARM 8 
+#define DEFAULT_PSO_G 0.3
+#define DEFAULT_PSO_L 0.6
+#define DEFAULT_PSO_R 0.1
+
+#ifdef XGEMM_EXEC
+extern bool tStrategyFlag;
+#endif
```

Comment:
No global variables if you can avoid them. This is tuner related, so this is not a good place anyway. I'm sure you can get rid of the global variable and the `ifdef` as well :-)